### ISSUE TITLE
Screen reader doesn't read total count and relative position of item in intellisense

### ISSUE THREAD

**Issue by shreedhart** [microsoft/vscode#88436]
- VSCode Version: 1.40.2
- OS Version: Windows 10 version 1903 (OS Build 18362.535)

**Steps to Reproduce**:

1. Open VS Code and create a new file.
2. Set language mode to Typescript.
3. Start Narrator or NVDA screen reader.
4. Type "String." in VS Code.

**Observed**: Screen reader reads "item arguments, docs, <full docs description of the item>"

**Expected**: Screen reader should also say the relative position and count of the item. eg. suggestion 1 of 13". This is standard with all list controls.

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This should be fixied with latest vscode insiders via https://github.com/microsoft/vscode/issues/3787
Let me know if you still see it with vscode insiders and I can reopen. Thanks
